lib/template.html

```diff
@@ -38,7 +38,12 @@
             playerBrandingId: "pleaseFillTheRightPlayerId",
             skin: {
                 inline: {
+                    startScreen: {
```

Comment:
These look like new additions as well. Was this intentional or intended for another PR?

Reply:
This is because I've branched from wrong branch as well. Sorry. Rolled back.